src/commands/powertune-lambda.js

```diff
@@ -195,6 +199,18 @@ PowertuneLambdaCommand.flags = {
 		description: "suppresses the prompt to show visualization",
 		default: false,
 		required: false
+	}),
+	autoOptimize: flags.boolean({
```

Comment:
@efimk-lu this was the change, to add two parameters to pass into the state machine